.travis.yml

```diff
@@ -14,17 +16,18 @@ cache:
   directories:
     - env/bin
     - env/lib/python2.7/site-packages
+    - node_modules
```

Comment:
There might've been a reason we took that out, I forget. I guess we'll find out again if we hit it again. :)